e2e/terraform/config/full-cluster/consul/aws.json

```diff
@@ -0,0 +1,4 @@
+{
+  "recursors": ["172.31.0.2"],
+  "retry_join": ["provider=aws tag_key=ConsulAutoJoin tag_value=auto-join"]
+}
```

Comment:
Note to reviewers: this is a full copy of the moved config now at `./e2e/terraform/config/dev-cluster/consul/aws.json`